en/micro-integrator/docs/references/connectors/kafka-connector/kafka-connector-producer-example.md

```diff
@@ -0,0 +1,120 @@
+# Kafka Connector Example
+
+Given below is a sample scenario that demonstrates how to send messages to a Kafka broker via Kafka topics.The publishMessages operation allows you to publish messages to the Kafka brokers via Kafka topics.
+
+## What you'll build
+Given below is a sample proxy service that illustrates how you can connect to a Kakfa broker with the `init` operation and then use the `publishMessages` operation to publish messages via the topic.
+
+**publishMessages**
+````xml
+<kafkaTransport.publishMessages>
+    <topic>topicName</topic>
+    <partitionNo>partitionNo</partitionNo>
+</kafkaTransport.publishMessages>
+````
+**Properties**
+
+| Property        | Description |
+| ------------- |-------------|
+| topic    | The name of the topic. |
+| partitionNo      | The partition number of the topic. |
+
+If required, you can add [custom headers](https://cwiki.apache.org/confluence/display/KAFKA/A+Case+for+Kafka+Headers) to the records in the `publishMessage` operation:
+
+````
+<topic.Content-Type>Value</topic.Content-Type>
+````
+You can add the parameter as follows in the publishMessage operation:
+
+````
+<kafkaTransport.publishMessage configKey="kafka_init">
+    <topic>topicName</topic>
+    <partitionNo>partitionNo</partitionNo>
+    <topicName.Content-Type>Value</topicName.Content-Type>
+</kafkaTransport.publishMessage>
+
+````
+The following diagram illustrates all the required functionality of the Kafka service that you are going to build.
+
+<img src="/assets/img/connectors/KafkaConnector.png" title="KafkaConnector" width="800" alt="KafkaConnector"/>
+
+## Configure the connector in WSO2 Integration Studio
+
+Follow these steps to set up the ESB Solution Project and the Connector Exporter Project.
+
+{!references/connectors/importing-connector-to-integration-studio.md!}
+
+1. Our project would look similar to the following (source view).
+
+    ```
+   <?xml version="1.0" encoding="UTF-8"?>
+   <proxy name="KafkaTransport" startOnLoad="true" transports="https http" xmlns="http://ws.apache.org/ns/synapse">
```

Comment:
As agreed, that we use an API instead of a proxy service? Then at testing section, we can use `curl`. 
SOAP is less used now. So our focus need to be always with APIs.